resources/sdk/purecloudjavascript/templates/index.d.ts.mustache

```diff
@@ -8,13 +9,13 @@ declare class ApiClientClass {
 	instance: ApiClientClass;
 	proxy: ProxyConfig;
 	superagent: any;
+	config: Configuration;
```

Comment:
Just double-check here. Should I add config here? This file is used in publishNPM script, not sure whether I should add it or not

Reply:
Yeah I think so, seems like it defines global properties